features/gezag-minderjarige/1.3b-is-geadopteerd-met-nederlandse-akte.feature

```diff
@@ -0,0 +1,137 @@
+      | burgerservicenummer | 000000036 |
+      Dan heeft de response een persoon met de volgende gegevens
+      | naam                | waarde    |
+      | burgerservicenummer | 000000036 |
+      En heeft de persoon een 'gezag' met de volgende gegevens
+      | naam                             | waarde                    |
+      | type                             | EenhoofdigOuderlijkGezag  |
+      | minderjarige.burgerservicenummer | 000000036                 |
+      | ouder.burgerservicenummer        | 000000012                 |
+
+    Scenario: Het kind is geadopteerd door twee ouders, de adoptie is als historisch gegeven opgenomen er is sprake van TweehoofdigOuderlijkGezag
+      Gegeven 'Carlo' is geadopteerd door 'Fabia' als ouder 1
+      En 'Carlo' is geadopteerd door 'Mohammed' als ouder 2
+      Gegeven persoon 'Carlo' 
+      * zijn de volgende gegevens gewijzigd
+      | naam                            | waarde     |
+      | aktenummer (81.20)              |            |
+      Als gezag wordt gezocht met de volgende parameters
+      | naam                | waarde    |
+      | burgerservicenummer | 000000036 |
+      Dan heeft de response een persoon met de volgende gegevens
+      | naam                | waarde    |
+      | burgerservicenummer | 000000036 |
+      En heeft de persoon een 'gezag' met de volgende gegevens
+      | naam                             | waarde                    |
+      | type                             | TweehoofdigOuderlijkGezag |
+      | minderjarige.burgerservicenummer | 000000036                 |
+      En heeft 'gezag' een 'ouder' met de volgende gegevens
+      | naam                | waarde    |
+      | burgerservicenummer | 000000012 |
+      En heeft 'gezag' een 'ouder' met de volgende gegevens
+      | naam                | waarde    |
+      | burgerservicenummer | 000000024 |
+
+  Regel: Kind is niet geadopteerd met nederlandse akte
+
+    Scenario: Het kind niet geadopteerd er is sprake van GezagNietTeBepalen
+      Als gezag wordt gezocht met de volgende parameters
+      | naam                | waarde    |
+      | burgerservicenummer | 000000036 |
+      Dan heeft de response een persoon met de volgende gegevens
+      | naam                | waarde    |
+      | burgerservicenummer | 000000036 |
+      En heeft de persoon een 'gezag' met de volgende gegevens
+      | naam        | waarde                                                                                                                                                                                                             |
+      | type        | GezagNietTeBepalen                                                                                                                                                                                                 |
+      | toelichting | gezag is niet te bepalen omdat minderjarige in het buitenland is geboren en geen Nederlandse adoptie-akte bekend is. |
```

Comment:
deze zit al in 1.3a toch?

Reply:
Dat klopt. Deze zit ook in 1.3a omdat 1.3b een vervolgvraag is op 1.3a. Vragen na 1.3a moeten ook ingevuld worden om tot een uitspraak te komen. Een uitspraak is gewenst, omdat er anders niks te verifiëren valt. Een fenomeen van "er moet altijd iets geverifieerd/getest zijn.

Reply:
Relevante discussie over het eventueel samenvoegen van v1.3a en v1.3b: https://github.com/BRP-API/brp-api-gezag/pull/207#discussion_r1834395308

Reply:
Conclusie van bovengenoemde discussie: voorlopig is het goed zo. Vragen 1.3a en 1.3b worden voorlopig niet samengevoegd.